packages/components/src/toggle-group-control/toggle-group-control/styles.ts

```diff
@@ -8,19 +8,26 @@ import styled from '@emotion/styled';
  * Internal dependencies
  */
 import { CONFIG, COLORS, reduceMotion } from '../../utils';
+import type { ToggleGroupControlProps } from '../types';
 
-export const ToggleGroupControl = css`
+export const ToggleGroupControl = ( {
+	size,
+}: {
+	size: NonNullable< ToggleGroupControlProps[ 'size' ] >;
+} ) => css`
 	background: ${ COLORS.ui.background };
 	border: 1px solid;
 	border-color: ${ COLORS.ui.border };
 	border-radius: ${ CONFIG.controlBorderRadius };
 	display: inline-flex;
-	min-height: ${ CONFIG.controlHeight };
```

Comment:
It looks like the CONFIG holds a few values for controls height:

https://github.com/WordPress/gutenberg/blob/e4252e430a04e0f528142a5a1f56e2b1971ecfce/packages/components/src/utils/config-values.js#L23-L27

As we work on standardizing the size variants, we should either:
- update these values and reference them as shared config across our components
- delete them from the shared config to avoid a scenario where some components have their height defined in "locally", and some other components reference a shared config that is out of sync

What do you think?

Reply:
> delete them from the shared config

Yes 🙈 I think this level of Logic & Order™ is hard to sustain unless they are accounted for at the design stage. Based on our current priorities, we can keep it simple for now.

Reply:
Even though I believe that updating the config and using across components would have have been a more clear way to enforce a common height across controls, I'm also ok with deleting the common config and using "local" styles in control components for their heights.

I had a quick look and there aren't too many usages left of `controlHeight`, so that deletion shouldn't be too hard:

![Screenshot 2022-07-13 at 18 05 50](https://user-images.githubusercontent.com/1083581/178779979-e049517e-6fb5-41d3-a955-e84573b3c5de.png)